_data/init-cli/docker_init.yaml

```diff
@@ -60,91 +62,104 @@ examples: |-
   This utility will walk you through creating the following files with sensible defaults for your project:
     - .dockerignore
     - Dockerfile
-    - docker-compose.yaml
```

Comment:
do we know if we really want to use this as the preferred filename? I know the spec describes it, but just wondering why we want to remove the prefix (I expect many more people will recognize `docker-compose.yaml` than just `compose.yaml`)

Reply:
If someone is starting a new project, isn't it best/preferred to use compose.yaml, since docker-compose.yaml is just supported for backwards compatibility?

Reply:
In principle, we don't anticipate users needing to care about the contents of these files. Just that when they run `docker compose up --build` it works. Also, we got user feedback that we (Docker) are being inconsistent in promoting `compose.yaml` and yet creating a `docker-compose.yaml` in a new product.